uploadNeuroDB/minc_insertion.pl

```diff
@@ -204,6 +216,11 @@
 print "\nlog dir is $LogDir and log file is $logfile \n" if $verbose;
 open LOG, ">>", $logfile or die "Error Opening $logfile";
 LOG->autoflush(1);
+# strings needed for the logHeader, if not set, as an argument, use empty string
+my $source_data_for_log = $tarchive;
```

Comment:
You are essentially doing 

`my $source_data_for_log = (defined $tarchive ? $tarchive :  "")`

which can be more concisely written as

`my $source_data_for_log = $tarchive // "";`

Same thing for the definition of variable `$upload_id_for_log`

Reply:
cool! Thanks!!